app/models/license.rb

```diff
@@ -408,7 +455,7 @@ class License < ApplicationRecord
     end
   }
   scope :banned, -> {
-    joins(:user).where.not(user: { banned_at: nil })
+    joins(:owner).where.not(users: { banned_at: nil })
```

Comment:
Should this be `owner:` to avoid ambiguity if also joined on `users`?

Reply:
Still have some ambiguity in the search methods, but not sure how I can work around that since Rails only aliases the join when your where clause includes a hash value, i.e. you can't do the following:

```rb
joins(:owner).where('owner.email ILIKE ?', '%example.com')
```

Instead, you have to do something like this:

```rb
joins(:owner).where.not(owner: { email: nil })
             .where('owner.email ILIKE ?', '%example.com')
```

I feel like you should be able to use `#references(:owner)` for this, but evidently that's not how it works.